tests/dummy/config/environment.js

```diff
@@ -6,6 +6,9 @@ module.exports = function(environment) {
     environment: environment,
     rootURL: '/',
     locationType: 'auto',
+    moment: {
+      includeLocales: ['ru']
```

Comment:
Change this to `includeLocales: true` and then you can remove the changes you made in _tests/dummy/app/controllers/public-pages/index.js_ and tests should be green.